src/app/ui/NewTaskForm.tsx

```diff
@@ -63,12 +63,14 @@ export const NewTaskForm = ({
     item: workflowStates[0],
     type: SelectorType.STATUS_SELECTOR,
   })
+
   const { renderingItem: _assigneeValue, updateRenderingItem: updateAssigneeValue } = useHandleSelectorComponent({
     item:
       filteredAssignees.find(
         (item) => item.id == filterOptions[FilterOptions.ASSIGNEE] || item.id == filterOptions[FilterOptions.TYPE],
       ) ?? null,
     type: SelectorType.ASSIGNEE_SELECTOR,
+    createTaskfieldUpdate: true,
```

Comment:
This is a quick and easy fix but perhaps we could make an `opts` param or something similar to pass `mode: 'createTaskFieldUpdate'`.

The reason for this is we are going to have to deal with both multi and single assignee selector in the near future, and this `opts` param could be used to pass additional params to `userHandleSelectorComponent` while still keeping the code clean.